app/src/main/res/values/strings.xml

```diff
@@ -1018,4 +1018,21 @@
     <string name="load_ingredient_detection_data_summary">No ingredient detection data has been found, try to load it from server</string>
     <string name="white_check_image_descr">A white check</string>
     <string name="hint_packaging">Examples: plastic, paper, glass</string>
+    <string name="check_first_time">check_first_time</string>
+
+    <!--ShowcaseView Strings-->
+    <string name="title_image_type">Select Image Type</string>
+    <string name="content_image_type">Use this to select the different image types</string>
```

Comment:
I would change "Use this" with "Press here", or something more specific than "use".